paparazzi/src/main/java/app/cash/paparazzi/Snapshot.kt

```diff
@@ -16,17 +16,27 @@
 package app.cash.paparazzi
 
 import com.squareup.moshi.JsonClass
+import dev.drewhamilton.poko.Poko
 import java.util.Date
 import java.util.Locale
 
 @JsonClass(generateAdapter = true)
```

Comment:
Yup, unfortunately Snapshot can't be internal, because it's a member of SnapshotHandler which is a public interface.  Is there a way to mark a JsonAdapter to be internal despite its "adapted class" being public?

Reply:
I just meant the adapter. If you ever change serialization framework, it'll be a breaking change, which sounds strange.

Reply:
I understand you meant the Adapter, but it seems like Moshi generates the adapter with the same access modifier as the backing class.  I'm all ears if there's a way to configure Moshi to generate an internal adapter.

Reply:
Hmm looks like I can copy the codegen'd adapter, mark it as internal and point to that mutated class, using [this](https://github.com/square/moshi/blob/2cc9ebb88e91ce8af690f1e9fa6fcb80167d41e1/moshi/src/main/java/com/squareup/moshi/JsonClass.kt#L66)?

Reply:
Sounds like a possibility, question is: is it worth it?
I'll defer to someone more experienced here (and/or a contributor to Moshi).